torchaudio/functional/functional.py

```diff
@@ -1666,7 +1698,24 @@ def pitch_shift(
     waveform_stretch = torch.istft(
         spec_stretch, n_fft=n_fft, hop_length=hop_length, win_length=win_length, window=window, length=len_stretch
     )
-    waveform_shift = resample(waveform_stretch, int(sample_rate / rate), sample_rate)
+    return waveform_stretch
+
+
+def _pitch_shift_postprocess(
```

Comment:
maybe call it `_fix_waveform_shape`. The basic function here is fix the length of waveform if it's longer than original or pad zeros if it's longer.

Reply:
+1. In general, names like pre-process / post-process should be avoided for implementation in library for the sake of clarity, unless the it's on the code is on application layer or it's really well-established, universally applicable and not change over time.